### PR TITLE
Adding platform arg to docker and pod man

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

Recent changes in ocp image create in quay, create only linux image, adding this platform arg will make those scripts workable also on macos
## 🎥 Demo

> Please add a video or an image of the behavior/changes

### CODE REVIEW

File: start-console-auth-mode.sh [hunk 1]
```diff
@@ -4,9 +4,9 @@ set -euo pipefail
 PATH+=:/usr/bin
 
 oc process -f scripts/oauth-client.yaml | oc apply -f -
-oc get oauthclient console-oauth-client -o jsonpath='{.secret}' > scripts/console-client-secret
-oc get secrets -n default --field-selector type=kubernetes.io/service-account-token -o json | \
-    jq '.items[0].data."ca.crt"' -r | python3 -m base64 -d > scripts/ca.crt
+oc get oauthclient console-oauth-client -o jsonpath='{.secret}' >scripts/console-client-secret
```
Comment (pcbailey) [inline start-console-auth-mode.sh:7]: Is there a reason why you're removing the spaces after the angle brackets?

Comment (metalice) [reply start-console-auth-mode.sh:7]: its the auto save.. didn't touch it just left prettier fixes

**Pull request merged**